components/application-authenticators/org.wso2.carbon.identity.application.authenticator.social/src/main/java/org/wso2/carbon/identity/application/authenticator/social/google/GoogleOAuth2AuthenticationConstant.java

```diff
@@ -22,7 +22,7 @@ private GoogleOAuth2AuthenticationConstant() {
     }
 
     public static final String GOOGLE_OAUTH_ENDPOINT = "GoogleOauthEndpoint";
-    public static final String GOOGLE_TOKEN_ENDPOINT = "GooglrTokenEndpoint";
+    public static final String GOOGLE_TOKEN_ENDPOINT = "GoogleTokenEndpoint";
     public static final String GOOGLE_USERINFO_ENDPOINT = "GoogleuserInfoEndpoint";
     public static final String GOOGLE_CONNECTOR_FRIENDLY_NAME = "Google OAuth2/OpenID Connect";
     public static final String GOOGLE_CONNECTOR_NAME = "GoogleOAUth2OpenIDAuthenticator";
```

Comment:
The constants and their values don't seem consistent.
public static final String GOOGLE_AUTHZ_ENDPOINT = "GoogleAuthzEndpoint";
public static final String GOOGLE_TOKEN_ENDPOINT = "GoogleTokenEndpoint";
public static final String GOOGLE_USERINFO_ENDPOINT = "GoogleUserInfoEndpoint";
     public static final String GOOGLE_CONNECTOR_FRIENDLY_NAME = "Google OAuth2/OpenID Connect";
     public static final String GOOGLE_CONNECTOR_NAME = "GoogleOIDCAuthenticator";